react/jest.config.cjs

```diff
@@ -25,8 +25,25 @@ module.exports = {
   collectCoverage: true,
 
   // An array of glob patterns indicating a set of files for which coverage information should be collected
```

Comment:
if we don't say what to grab, it only shows coverage for files that are tested.  so if we aren't testing a component then it doesn't show up in our coverage.